app.yaml

```diff
@@ -10,6 +10,8 @@ threadsafe: yes
 
 # Handlers define how to route requests to your application.
 handlers:
+- url:/.*
+  script: scrape_all.py
```

Comment:
add `login: admin` so not everyone can hit it

https://cloud.google.com/appengine/docs/python/config/cron?hl=en